packages/client/src/features/context-menu/di.config.ts

```diff
@@ -13,10 +13,9 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import { bindAsService } from '@eclipse-glsp/protocol';
 import { ContainerModule } from 'inversify';
-import { ContextMenuProviderRegistry, IContextMenuService } from 'sprotty';
-import { TYPES } from '../../base/types';
+import { ContextMenuProviderRegistry, IContextMenuService, bindAsService } from '~glsp-sprotty';
+import { TYPES } from '../../glsp-sprotty/types';
```

Comment:
nit: should be added to the imports of `~glsp-sprotty`
(applies to all subsequent imports as well, I just don't want to push too many comments for the same thing ;-))